pkg/cli/project/set-current.go

```diff
@@ -0,0 +1,66 @@
+package project
+
+import (
+	"fmt"
+
+	"github.com/fuseml/fuseml-core/pkg/cli/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// SetCurrentOptions holds the options for 'project set' sub command
+type SetCurrentOptions struct {
+	global *common.GlobalOptions
+	format *common.FormattingOptions
+	Name   string
+}
+
+// NewSetCurrentOptions creates a ProjectSetCurrentOptions struct
+func NewSetCurrentOptions(o *common.GlobalOptions) *SetCurrentOptions {
+	res := &SetCurrentOptions{global: o}
+	res.format = common.NewSingleValueFormattingOptions()
+	return res
+}
+
+// NewSubCmdProjectSetCurrent creates and returns the cobra command for the `project set` CLI command
+func NewSubCmdProjectSetCurrent(gOpt *common.GlobalOptions) *cobra.Command {
+
+	o := NewSetCurrentOptions(gOpt)
+
+	cmd := &cobra.Command{
+		Use:   `set-current {-n|--name NAME}`,
+		Short: "Set current project.",
+		Long:  `Set project as a current one`,
+		Run: func(cmd *cobra.Command, args []string) {
+			common.CheckErr(o.validate())
+			common.CheckErr(o.run())
+		},
+		Args: cobra.ExactArgs(0),
+	}
+
+	cmd.Flags().StringVarP(&o.Name, "name", "n", "", "project name")
+	o.format.AddSingleValueFormattingFlags(cmd, common.FormatYAML)
+	cmd.MarkFlagRequired("name")
+	return cmd
+}
+
+func (o *SetCurrentOptions) validate() error {
+	return nil
+}
+
+func (o *SetCurrentOptions) run() error {
+
+	if viper.GetString("CurrentProject") == o.Name {
+		fmt.Printf("Project %s already marked as current.\n", o.Name)
+		return nil
+	}
+
+	viper.Set("CurrentProject", o.Name)
+
+	if err := viper.WriteConfig(); err != nil {
+		return err
+	}
+
+	fmt.Printf("Project '%s' saved as current.\n", o.Name)
```

Comment:
This wording sounds a bit weird to me, as a suggestion I would rephrase this something like `Current project set to %s`.

Side note: IMO, using `default` (as is on the PR title) instead of `current` would fit better, but I do not want to ask you to rewrite everything using `default` instead of `current`.

Reply:
You are right about wording. 

I'm not sure about the other part